RELEASE_NOTES.rst

```diff
@@ -1,3 +1,14 @@
+coala-bears 0.9.2
+=================
+- A dependency problem in `RadonBear` causing install failure has been fixed
+  `Issue #1228 <https://github.com/coala/coala-bears/issues/1228>`_
+- coala depedency has been fixed to only allow 0.9 series coala install with
```

Comment:
:point_up: [January 3, 2017 5:13 PM](https://gitter.im/coala/coala?at=586bbf9e9d4cc4fc536fe49b)
three? i think its four, because heres "dependency" :D